packages/nartus_ui_package/lib/widgets/google_map.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:ui' as ui;
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:nartus_ui_package/nartus_ui.dart';
+
+class GoogleMapWidget extends StatefulWidget {
+  final double _latitude;
+  final double _longitude;
+  final bool? _isAnimation;
+  final void Function()? _onTap;
```

Comment:
please change the name of this `_onTap` to be more specific, like `_onMarkerTap`, which takes in markerId param, so that it can be reused later as we have more markers